sigs/build/community-builds.md

```diff
@@ -0,0 +1,76 @@
+# Community Supported TensorFlow Builds and Releases
+
+
+## Overview
+
+TensorFlow is used in many more environments and configurations than is practical for the core team to regularly test and support: so we need a way to include federated third party testing and builds.
+
+This document describes a process for creating third party builds of TensorFlow, federating tests and builds, and making the build artifacts available to users. Examples of such builds include those optimized for particular hardware configurations, operating system environments, or other specific applications.
+
+There are three major phases of the process:
+
+
+
+1.  Engagement — connect with the TensorFlow core team and work on a plan for integration, tests, documentation and support.
```

Comment:
either lose the period at the end, or add a period to 2. and 3. below. (sorry, my original error.)